11ty/definitions/xenophobia.md

```diff
@@ -0,0 +1,22 @@
+---
+title: xenophobia
+slug: xenophobia
+defined: true
+excerpt:  hatred of and discrimination against people from other countries, their culture, and/or their religion that is rooted in the fear of people who are "different"
+speech: noun
+flag:
+  level: warning
+  text: content warning
+---
+
+hatred of and discrimination against people from other countries, their culture, and/or their religion that is rooted in the fear of people who are "different"
+
+## Issues
+
+Xenophobia when broken down translates to the "fear of strangers." It stems from a psychological point of view that there is an "ingroup" and an "outgroup" that exist within a community, which is known as [ingroup-outgroup bias](http://psychology.iresearchnet.com/social-psychology/group/ingroup-outgroup-bias/). Whether it's conscious or unconscious, ingroup-outgroup bias can lead to harmful results, including xenophobia.
+
+## Impact
+
+Xenophobia can perpetuate physical, emotional, and psychological harm to those who look "different," which usually include people of color, immigrants, and minorities of a country. This can also extend harm to those who look similar to those that are perceived as "different." Xenophobia can further result in implicit bias, harmful public policies, hate crimes, and genocide.
```

Comment:
```suggestion
Xenophobia can perpetuate physical, emotional, and psychological harm to those who look/act/believe "different," which usually include people of color, immigrants, and minorities of a country. This can also extend harm to those who look similar to those that are perceived as "different." Xenophobia can further result in implicit bias, harmful public policies, hate crimes, and genocide.
```
```suggestion
Xenophobia can perpetuate physical, emotional, and psychological harm to those who look/believe/behave "differently," which usually include people of color, immigrants, and minorities of a country. This can also extend harm to those who look similar to those that are perceived as "different." Xenophobia can further result in implicit bias, harmful public policies, hate crimes, and genocide.
```
I think here we need to also paint the picture of different _from what_. What's being implicitly said here is there is a "right way" to act and failure to conform will lead to consequence.